src/Common/OptimizedRegularExpression.cpp

```diff
@@ -540,7 +540,7 @@ bool OptimizedRegularExpressionImpl<thread_safe>::match(const char * subject, si
             }
         }
 
-        return re2->Match(StringPieceType(subject, subject_size), 0, subject_size, RegexType::UNANCHORED, nullptr, 0);
+        return re2->Match(absl::string_view(subject, subject_size), 0, subject_size, RegexType::UNANCHORED, nullptr, 0);
```

Comment:
Why `absl::string_view` and not `std::string_view`?

Reply:
Of course, both are equivalent in post-C++-17. I found usage of `absl` in our code a bit more natural as that is what re2 uses everywhere in its API. Once they dropped their absl dependency, we can move over to `std`.

Reply:
We already have `StringRef` we are trying to replace with `std::string_view`, why not just use `std::string_view` here also instead of introducing a third class with same functionality?

Reply:
I don't mind too much, `absl::string_view` and `std::string_view` are aliases. Okay, let me use `std::` then.